### DISCUSSION TITLE
How to import less Mixin  in vanilla project?

### DISCUSSION THREAD

**Discussion by ziven27** [codesandbox/codesandbox-client#5789] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/7261176/118397202-7da80100-b685-11eb-909f-5f1ff814113a.png)

I import `./mixin.less`  in `styles.less` file but it dose not works.

it is keep `Transpiling Modules...` all the time and feels like it will never stop.

**Comment by DeMoorJasper** [comment, marked as answer]
This is a bug, fixed by https://github.com/codesandbox/codesandbox-client/pull/5811
